packages/api/srcbook/path.mts

```diff
@@ -0,0 +1,22 @@
+import Path from 'node:path';
```

Comment:
While this doesn't add a ton, I want a single source of truth for this. Feels error prone otherwise and should make future refactors of path related changes easier to find/replace than `Path.join`.

Reply:
I like it, especially if we want to map dir IDs later